docs/docs/concepts/persistence.md

```diff
@@ -216,6 +216,106 @@ The final thing you can optionally specify when calling `update_state` is `as_no
 
 ![Update](img/persistence/checkpoints_full_story.jpg)
 
+## Memory Store
```

Comment:
i think this is missing some description of what is returned from `store.search`. eg we access a `value` attribute... what??? what is that?